pbj-core/pbj-runtime/src/test/java/tests/ComparableOneOfTest.java

```diff
@@ -46,9 +48,19 @@ void equalsWorks() {
         assertEquals(false, oneOf.equals(TestEnum.KIND1));
     }
 
-    public enum TestEnum {
+    public enum TestEnum implements EnumWithProtoMetadata {
         KIND1,
-        KIND2
+        KIND2;
+
+        @Override
+        public int protoOrdinal() {
+            return ordinal();
```

Comment:
The implementation of `protoOrdinal()` should be more complex IMO, resulting in negative values and gaps.

Reply:
This is a test implementation and we currently don't have any tests that would benefit from a more complex implementation here.